src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Hearing.java

```diff
@@ -19,6 +20,16 @@ public Hearing(@JsonProperty("value") HearingDetails value) {
 
     @Override
     public int compareTo(Hearing o) {
-        return value.getHearingDateTime().compareTo(o.getValue().getHearingDateTime());
+
+        int idCompare = 0;
+
+        if (!StringUtils.isEmpty(o.getValue().getHearingId()) && !StringUtils.isEmpty(value.getHearingId())) {
+            idCompare = o.getValue().getHearingId().compareTo(value.getHearingId());
+        }
+
+        if (idCompare != 0) {
+            return -1 * idCompare;
+        }
+        return -1 * o.getValue().getHearingDateTime().compareTo(value.getHearingDateTime());
```

Comment:
does [this](https://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/builder/CompareToBuilder.html#reflectionCompare-java.lang.Object-java.lang.Object-boolean-) not work?